### ISSUE TITLE
[Feat] keep empty line after HashBang

### ISSUE THREAD

**Issue by Airkro** [IanVS/prettier-plugin-sort-imports#183]
**Is your feature request related to a problem?**
Please describe a clear and concise description of what the problem is. E.g. I'm always frustrated when [...]

Currently, this plugin will remove empty line after HashBang.

```diff
  #!/usr/bin/env node
-                               
  import foo from 'bar';
```

**Describe the solution you'd like**

- Keep an empty line after HashBang, if it exists.

**Describe alternatives you've considered**

- Enforce an empty line after HashBang.

**Additional context**

**Comment by fbartho** [comment]
I personally don’t see a problem with this feature change.

I think we already have a special case for shebang-lines, so it’s probably a small tweak to adjust this code.

The biggest problem is deciding if this is a breaking change from a versioning perspective.

**Comment by IanVS** [comment]
I think it could be considered a "bugfix", though we might want to release in a minor.  Even prettier itself allows formatting changes between versions.  We should make our stance explicit in the README, though...

**Comment by Airkro** [comment]
Add a new option to enable that feature, can avoid breaking change.

**Comment by IanVS** [comment]
Yes, but that's how we ended up with a bajillion options previously, and we worked hard to remove them.  Prettier itself is opinionated and has very few options, and we want to follow that same philosophy.
